libs/community/langchain_community/document_loaders/pebblo.py

```diff
@@ -45,6 +46,7 @@ def __init__(
         classifier_url: Optional[str] = None,
         *,
         classifier_location: str = "local",
+        **kwargs: Any,
```

Comment:
Ideally, while defining any function, `* and **` should come at end

Reply:
`*` is used to indicate the beginning of `keyword-only arguments` in the method signature, so it cannot be placed at the end.

It looks like this was suggested by the Langchain reviewer on [this PR](https://github.com/langchain-ai/langchain/pull/22565#discussion_r1649121450).

Reply:
okay